### PR TITLE
Fix for linking errors of iOS framework from common-all module.

### PR DESCRIPTION
**Pull request by OlexandrStepanov**
These changes fixes linkDebugFrameworkIos task in the build process.

### CODE REVIEW

**Review by RubyLichtenstein — APPROVED** [review]

**Comment by OlexandrStepanov** [comment]
@RubyLichtenstein , should I merge this or you should? If me - I can't see the merge button.

**Comment by RubyLichtenstein** [comment]
Sorry, I thought that you have merge button, I'll merge it, thank you :)

**Pull request merged**
